docs/design/service_traffic_policy.md

```diff
@@ -93,7 +93,7 @@ cookie=0xdeff105, duration=3189.787s, table=1, n_packets=108, n_bytes=23004, pri
 3. In the host, we have an IPtable rule in the PREROUTING chain that DNATs this packet matched on nodePort to a masqueradeIP (169.254.169.3) used specially for this traffic flow.
 
 ```
-[3:180] -A OVN-KUBE-NODEPORT -p tcp -m addrtype --dst-type LOCAL -m tcp --dport 31746 -j DNAT --to-destination 169.254.169.3:31746
+[3:180] -A OVN-KUBE-ETP -p tcp -m addrtype --dst-type LOCAL -m tcp --dport 31746 -j DNAT --to-destination 169.254.169.3:31746
```

Comment:
This commit doesn't have anything to do with implementing ITP. I would prefer we move it to its own PR.

Reply:
good point.... I was having an internal debate with myself if I should do that... will move this to new PR.

Reply:
I will keep that PR dependent on this since some of the code cleanup in that 3rd commit is better and I'd like that to go to 4.10.z so don't want to move those parts to ITP PR since this won't be backported